packages/network/lib/uri.ts

```diff
@@ -46,7 +46,10 @@ export function removePort (urlObject) {
 
   // set host to undefined else url.format(...) will ignore the port property
   // https://nodejs.org/api/url.html#url_url_format_urlobject
+  // Additionally, the types are incorrect (don't include undefined), so we add TS exceptions
+  /* @ts-ignore */
```

Comment:
Is this a case where we can use `ts-expect-error`?

Reply:
We could, but it feels like an odd scenario since you're fighting an incorrect type.